seqr/management/tests/reload_clinvar_all_variants_tests.py

```diff
@@ -0,0 +1,310 @@
+            # Case 1: Missing AlleleId in <SimpleAllele>
+            ("", 'Assembly="GRCh38" Chr="1" variantLength="1" positionVCF="1" referenceAlleleVCF="G" alternateAlleleVCF="A"'),
+
+            # Case 2: Missing alternateAlleleVCF in <SequenceLocation>
+            ('AlleleID="5603"', 'Assembly="GRCh38" Chr="1" variantLength="1" positionVCF="1" referenceAlleleVCF="G"'),
+        ]:
+            data = f'''<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+            <ClinVarVariationRelease xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                                     xsi:noNamespaceSchemaLocation="http://ftp.ncbi.nlm.nih.gov/pub/clinvar/xsd_public/ClinVar_VCV_2.4.xsd"
+                                     ReleaseDate="2025-06-30">
+                <VariationArchive></VariationArchive>
+                <VariationArchive>
+                    <ClassifiedRecord>
+                        <SimpleAllele {simple_allele_attrs}>
+                        </SimpleAllele>
+                    </ClassifiedRecord>
+                </VariationArchive>
+                <VariationArchive>
+                    <ClassifiedRecord>
+                        <SimpleAllele {simple_allele_attrs}>
+                            <Location>
+                                <SequenceLocation {sequence_location_attrs}/>
+                            </Location>
+                        </SimpleAllele>
+                        <Classifications>
+                            <GermlineClassification>
+                                <Description>Pathogenic</Description>
+                                <ReviewStatus>criteria provided, conflicting classifications</ReviewStatus>
+                            </GermlineClassification>
+                        </Classifications>
+                    </ClassifiedRecord>
+                </VariationArchive>
+                <VariationArchive>
+                    <ClassifiedRecord>
+                        <SimpleAllele {simple_allele_attrs}>
+                            <Location>
+                                <SequenceLocation {sequence_location_attrs}/>
+                            </Location>
+                        </SimpleAllele>
+                        <Classifications>
+                            <GermlineClassification>
+                                <!-- No Description -->
+                                <ReviewStatus>criteria provided, conflicting classifications</ReviewStatus>
+                            </GermlineClassification>
+                        </Classifications>
+                    </ClassifiedRecord>
+                </VariationArchive>
+            </ClinVarVariationRelease>'''
+            responses.add(
+                responses.GET,
+                WEEKLY_XML_RELEASE,
+                status=200,
+                body=gzip.compress(data.encode()),
+                stream=True,
+            )
+            call_command('reload_clinvar_all_variants')
+
```

Comment:
we should actually test. that the results of this edge case are as expected, not just that it succeeds. I think just asserting that the model count is as expected would be sufficient